frontend/src/_helpers/utils.js

```diff
@@ -34,15 +34,15 @@ export function resolveAll(data, state) {
 
 }
 
-export function resolveReferences(object, state) {
+export function resolveReferences(object, state, customObjects) {
   if (typeof object === 'string') {
     if (object.startsWith('{{') && object.endsWith('}}')) {
       const code = object.replace('{{', '').replace('}}', '');
```

Comment:
This can also be written as 
```suggestion
      const code = object.replace(/^{{|}}$/g,'')
```

Reply:
Yes, but I think the current version is easier to read.